index.js

```diff
@@ -2,9 +2,40 @@ const express = require('express')
 const path = require('path')
 const PORT = process.env.PORT || 5000
 
+const mockdb = require("./mockdb")
+var db = new mockdb.mockDb(mockdb.availableItems)
+
+function searchItems(req, res) {
+  tlSplit = req.query.topLeftLocation ?
+    req.query.topLeftLocation.split(",")
+    : [];
+  lrSplit = req.query.lowerRightLocation ?
+    req.query.lowerRightLocation.split(",")
+    : [];
+
+  if (tlSplit.length != 2 || lrSplit.length != 2) {
+    res.status(400).send({
+      message: 'Invalid location data.'
```

Comment:
Perhaps add some information how valid location data should look like

Reply:
I resolved this in #21 .